src/Sarif/CodeGenHints.json

```diff
@@ -1,12 +1,64 @@
 {
+  "annotatedCodeLocation": [
+    {
+      "kind": "BaseTypeHint",
+      "arguments": {
```

Comment:
The code generator now supports a "base type" hint, which allows the classes with properties to derive from `PropertyBagHolder`.